sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/Catalogs.java

```diff
@@ -48,6 +48,7 @@ private Catalogs() {
    * @return an initialized CatalogPlugin
    * @throws CatalogNotFoundException if the plugin class cannot be found
    * @throws SparkException if the plugin class cannot be instantiated
+   * @since 3.0.0
```

Comment:
This is a private class (`@Private`)

Reply:
If it doesn't work, we can rewrite in with Scala and use `private[sql]`.

Reply:
Thanks, rewrite done in 69ab608.